src/month.rs

```diff
@@ -167,23 +167,7 @@ impl Month {
     ///
     /// Yields `None` if `year` is out of range for `NaiveDate`.
     pub fn num_days(&self, year: i32) -> Option<u8> {
-        Some(match *self {
```

Comment:
I definitely want to keep this logic in `Month`.

Reply:
moving it here just avoids an `unwrap` call but I can move it back if you'd really prefer